styles/global.scss

```diff
@@ -82,23 +84,26 @@ strong > a, a > strong {
   h2 {
     font-size: 16px;
   }
-  
+
   p {
     font-size: 12px;
   }
-  
-  a > strong, strong > a {
+
+  a > strong,
+  strong > a {
     margin: 0 auto;
     display: block;
     width: 120px;
   }
-} 
+}
+
 :global {
   .section-content {
     flex: 1;
     display: flex;
     width: 100%;
     max-width: $max-desktop-size;
+    margin: auto;
```

Comment:
This is the only line added to this file. It doesn't seem to affect the style of other parts. This is used to center the content. It's also what bootstrap does to center content: https://www.w3schools.com/bootstrap4/tryit.asp?filename=trybs_gs_container2&stacked=h

![image](https://user-images.githubusercontent.com/6027291/94820063-6528f600-03d6-11eb-9fc1-466939b38e32.png)